src/backend/marsha/bbb/permissions.py

```diff
@@ -80,3 +80,24 @@ def has_permission(self, request, view):
             playlist__classrooms__id=view.get_object_pk(),
             user__id=request.user.id,
         ).exists()
+
+
+class IsClassroomAdmin(permissions.BasePermission):
```

Comment:
```suggestion
class IsClassroomPlaylistOrOrganizationAdmin(permissions.BasePermission):
```

I think the permission name should be explicit. WDYT ?

Reply:
It's better, yes :+1: